### DISCUSSION TITLE
Error: Unable to find any supported Python versions.

### DISCUSSION THREAD

**Discussion by chandrakeshram** [vercel/community#6328] [Help] [answered]
### Page to Investigate

https://vercel.com/chandrakeshram7s-projects/crop-predict/8G19kTmoJxRLEQz45h8CbxRuwLS9

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/79987328/4e200ea0-1a39-4daa-99fd-ca5492d8d01a)
I have a simple python api that has some hardcoded values and will provide me the recommendation of crops. When i had deployed one api earlier it got deployed very easily but i dont know why it is happening now. Please help me out
![image](https://github.com/vercel/community/assets/79987328/1eaec4f6-e467-4557-acf4-1d215ba52a2d)
This is requirements.txt file 
![image](https://github.com/vercel/community/assets/79987328/eb909f46-a816-46d4-b3e2-b6b73b85217c)
This is vercel.json 
Please help me out and Thanks in advance

**Comment by swarnava** [comment, marked as answer]
Can you go to Project Settings and Make sure Node 18.x is selected?
